pkgs/development/tools/misc/dlang-dfmt/mkDub.nix

```diff
@@ -0,0 +1,141 @@
+{ pkgs ? import <nixpkgs> { }, stdenv ? pkgs.stdenv, lib ? pkgs.lib
+, dtools ? pkgs.dtools or pkgs.rdmd, dmd ? pkgs.dmd, dcompiler ? dmd
+, dub ? pkgs.dub }:
+
+with stdenv;
```

Comment:
Filewide `with` statements like this can easily cause confusion, as it can expose quite a lot of symbols into the current namespace. It's not obvious looking at the functions below where they come from. Let's remove it, and rather use `stdenv.*` whenever you need to use something from it.